fractal_tasks_core/tasks/projection.py

```diff
@@ -99,28 +113,30 @@ def projection(
     new_image = new_ngff_image.get_image()
 
     # Process the image
-    z_axis_index = orginal_image.find_axis("z")
-    source_dask = orginal_image.get_array(
-        mode="dask", preserve_dimensions=True
-    )
-
+    source_dask = orginal_image.get_array(mode="dask")
     dest_dask = method.apply(dask_array=source_dask, axis=z_axis_index)
     dest_dask = da.expand_dims(dest_dask, axis=z_axis_index)
     new_image.set_array(dest_dask)
     new_image.consolidate()
     # Ends
 
     # Copy over the tables
```

Comment:
The flow here is that all tables were copied over, but now we modify the ROI tables to project them, right? Given that copy_tables above is set to True.
If so, let's update the comment here to specify that